Vagrantfile

```
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  
+# box settings
+  config.vm.box = "ubuntu/focal64"
+
+
+
+  #providor settings
+
+  config.vm.provider "virtualbox" do |vb|
+  vb.memory = 2048
+  vb.cpus = 4  
+  end
+
+ 
+# network settings
+
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  #config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+  #config.vm.network "private_network", ip: "192.168.33.10"
+  # config.vm.network "public_network"
+
+# folder settings
+
+  #config.vm.synced_folder ".", "/var/webapp"
+
+
+# provision settings
+
+ config.vm.provision "shell", inline: <<-SHELL
+  sudo apt-get update
+  sudo apt install python3.8-venv -y
+  sudo apt install python3-pip -y
+  sudo apt install curl
+  curl -LJO https://raw.githubusercontent.com/georgef21/Flask-vagrant-project/main/webapp.py
+  curl -LJO https://raw.githubusercontent.com/georgef21/Flask-vagrant-project/main/deployment
```

This does not install the resources there is no repo named 'Flask-vagrant-project', you need to change the URL here and for webapp.py
```suggestion
  curl -LJO https://raw.githubusercontent.com/georgef21/vagrant-flask-exercise/main/deployment
```

---

@georgef21 this fails for me too. Please fix

---

Actually you don't even need to download anything. The application files are right here in the repo next to Vagrantfile.

---

just use shell provisioner with `path` and not `inline`